src/index.js

```diff
@@ -0,0 +1,120 @@
+import './style.css';
+import { Todo, TodoLoad } from './Add_remove.js';
+import { IsChecked, clearAllCompleted } from './completed.js';
+
+let todos = [];
+
+const MyTodoList = document.getElementById('TodoList');
```

Comment:
`[optional]` Based on the general feedback I provided at the top of this pull request, please use camelcase notation for all variable and function names. For example, instead of `MyTodoList`, use `myTodoList`. This will help to improve the readability and maintainability of our code.